vochain/state.go

```diff
@@ -664,34 +679,34 @@ func (v *State) EnvelopeList(processID []byte, from, listSize int,
 }
 
 // Header returns the blockchain last block committed height
-func (v *State) Header(isQuery bool) *models.TendermintHeader {
-	v.RLock()
-	headerBytes, err := v.mainTreeViewer(isQuery).Get(headerKey)
-	v.RUnlock()
-	if err != nil {
-		log.Fatalf("cannot get headerKey from mainTree: %s", err)
-	}
-	var header models.TendermintHeader
-	if err := proto.Unmarshal(headerBytes, &header); err != nil {
-		log.Fatalf("cannot get proto.Unmarshal header: %s", err)
-	}
-	return &header
-}
+// func (v *State) Header(isQuery bool) *models.TendermintHeader {
+// 	v.Tx.RLock()
+// 	headerBytes, err := v.mainTreeViewer(isQuery).Get(headerKey)
+// 	v.Tx.RUnlock()
+// 	if err != nil {
+// 		log.Fatalf("cannot get headerKey from mainTree: %s", err)
+// 	}
+// 	var header models.TendermintHeader
+// 	if err := proto.Unmarshal(headerBytes, &header); err != nil {
+// 		log.Fatalf("cannot get proto.Unmarshal header: %s", err)
+// 	}
+// 	return &header
+// }
```

Comment:
just remove it? IMO leaving uncommented code isn't helpful, because we can always retrieve it from git history